.gitignore

```diff
@@ -180,8 +180,8 @@ dist/preview release/package/
 /Viewer/dist/viewer.js
 /Viewer/dist/viewer.min.js
 dist/preview release/viewer/babylon.d.ts
-dist/preview release/viewer/babylonjs.loaders.d.ts
-dist/preview release/viewer/babylon.glTF2Interface.d.ts
+dist/preview release/viewer/loaders.d.ts

```

Comment:
why not babylon.* ?

Reply:
i thought we remove the babylon. . [modulename].* , this is what we discussed

Reply:
Well misunderstanding then :)  it must be babylon.modulename.* (I think the reason of the misunderstanding is that all modules should be babylon.* for me but the viewer is not following this rule)